ccmlib/node.py

```diff
@@ -803,6 +803,8 @@ def run_cli(self, cmds=None, show_output=False, cli_options=[]):
 
     def run_cqlsh(self, cmds=None, show_output=False, cqlsh_options=[], return_output=False):
         cqlsh = self.get_tool('cqlsh')
+        if not isinstance(cqlsh, list):
```

Comment:
had to change `get_tool` to return a list of arguments, so `docker exec` can be added